server/pulp/server/managers/content/query.py

```diff
@@ -290,7 +290,14 @@ def request_content_unit_file_path(self, content_type, relative_path):
         unit_path = os.path.join(self.get_root_content_dir(content_type), relative_path)
         unit_dir = os.path.dirname(unit_path)
         if not os.path.exists(unit_dir):
-            os.makedirs(unit_dir)
+            try:
+                os.makedirs(unit_dir)
+            except OSError, e:
+                # In case two workers are syncing exact same content simultaneously and the
```

Comment:
Consider removing this comment. Here are some reasons.

* It's linked in github to this PR and in Redmine using the ref, so it would be easy for a developer to find issue #902 and this PR for these lines if they ever have questions.
* More generally, we could fill the whole codebase with comments like these which need to be maintained over time and could potentially double the code size.
* If comments ever don't get maintained they are really unhelpful. Consider that this codepath may eventually not be called by two workers at the same time, and then this comment would suggest that they do.
* Comments should be reserved for really, really non-obvious programmer intent. I think this being race condition free is obvious.